release-notes/5.0/api-diff/netstandard2.1/5.0_System.Text.Json.md

```diff
@@ -161,21 +161,30 @@
 +        public static byte[] SerializeToUtf8Bytes(object value, Type inputType, JsonSerializerOptions options = null);
 +        public static byte[] SerializeToUtf8Bytes<TValue>(TValue value, JsonSerializerOptions options = null);
 +    }
++    public enum JsonSerializerDefaults {
++        General = 0,
++        Web = 1,
++    }
 +    public sealed class JsonSerializerOptions {
 +        public JsonSerializerOptions();
++        public JsonSerializerOptions(JsonSerializerDefaults defaults);
 +        public JsonSerializerOptions(JsonSerializerOptions options);
 +        public bool AllowTrailingCommas { get; set; }
 +        public IList<JsonConverter> Converters { get; }
 +        public int DefaultBufferSize { get; set; }
++        public JsonIgnoreCondition DefaultIgnoreCondition { get; set; }
 +        public JsonNamingPolicy DictionaryKeyPolicy { get; set; }
 +        public JavaScriptEncoder Encoder { get; set; }
 +        public bool IgnoreNullValues { get; set; }
++        public bool IgnoreReadOnlyFields { get; set; }
 +        public bool IgnoreReadOnlyProperties { get; set; }
++        public bool IncludeFields { get; set; }
 +        public int MaxDepth { get; set; }
++        public JsonNumberHandling NumberHandling { get; set; }
 +        public bool PropertyNameCaseInsensitive { get; set; }
 +        public JsonNamingPolicy PropertyNamingPolicy { get; set; }
 +        public JsonCommentHandling ReadCommentHandling { get; set; }
-+        public ReferenceHandling ReferenceHandling { get; set; }
++        public ReferenceHandler ReferenceHandler { get; set; }
```

Comment:
@Anipik this was actually an API change between 5.0 previews, why is it showing here? 
Plus, removal of `ReferenceHandling` doesn't even appear anywhere else in this PR.

Reply:
I believe this is a diff of a diff. So you’re seeing the old name disappear and the correct name be added

Reply:
Ah right. Thanks for clarifying.